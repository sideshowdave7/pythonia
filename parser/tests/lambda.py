a = lambda:print("Hello world")
b = lambda : lambda : lambda : print("hi")
c = lambda x, y, z : lambda a, b, c : a()
