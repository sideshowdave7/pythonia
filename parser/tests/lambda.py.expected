(program
 (= (a) (lambda () (indexed print (called "Hello world"))))
 (= (b) (lambda () (lambda () (lambda () (indexed print (called "hi"))))))
 (= (c) (lambda (x y z) (lambda (a b c) (indexed a (called))))))

